### ISSUE TITLE
license for the operator

### ISSUE THREAD

**Issue by nyrahul** [openshift/file-integrity-operator#232]
It would be good if the license for the file-integrity-operator could be explicitly specified.
Under the hood, AIDE uses GPLv2 and thus the operator license needs to be aligned with this.

**Comment by mrogers950** [comment]
I agree it's not clear; currently our files say Apache 2.0, which was the default boilerplate generated by operator-sdk. I'm not opposed to evaluate switching to GPLv2, but note that we only call on the AIDE process that is installed to the base image layer, so we shouldn't need to explicitly align with AIDE's license. 

@openshift/infrastructure-security-compliance + @JAORMX thoughts/opinions on license?

**Comment by JAORMX** [comment]
While AIDE gets run under the hood, the operator executes it as a binary and does not link to it. So FIO can indeed stay using the current Apache license. Switching to GPLv2 might actually be problematic for FIO since we have dependencies that are most likely non-conformant with GPL... and golang dependencies are statically linked, so the license becomes stricter. I'd say staying in Apache is the safer bet and our current usage of AIDE (through calling the binary as a separate process) does not violate GPLv2.
